controllers/controllervm_controller.go

```diff
@@ -267,15 +271,12 @@ func (r *ControllerVMReconciler) networkCreateAttachmentDefinition(instance *osp
 	}
 
 	// Generate the Network Definition objects
-	for _, v := range instance.Spec.Networks {
-		data.Data["Network"] = v.Name
-		manifests, err := bindatautil.RenderDir(filepath.Join(ManifestPath, "network"), data)
-		if err != nil {
-			r.Log.Error(err, "Failed to render network manifests : %v")
-			return err
-		}
-		objs = append(objs, manifests...)
+	manifests, err := bindatautil.RenderDir(filepath.Join(ManifestPath, "network"), data)
+	if err != nil {
+		r.Log.Error(err, "Failed to render network manifests : %v")
+		return err
 	}
+	objs = append(objs, manifests...)
```

Comment:
Just a question. Why did you move the append for objs here and comment them out in the loop below?

Reply:
the objs here is the same as before. for now we don't have an array of networks in the CRD, so no loop. The ops bellow were just not required as we don't need to add the modified obj again to objs. I missed to delete to commented like completely